api/config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  scope '/api' do
+  scope :api do
```

Comment:
I want to know why using a symbol is better. Could you please explain me?

Reply:
The symbols are immutables, the immutable nature of symbols makes them very valuable in programming because mutable objects can cause bugs that are difficult to detect.

Reply:
Thanks for your explanation